roscube-i/nvidia_driver.rst

```diff
@@ -0,0 +1,102 @@
+.. _nvidia_driver:
+
+How to Install NVDIA Driver
+###########################
+
+The tutorial will guide you how to install nvidia driver.
+
+Supported Model: ROScube-I series
+
+1. Update your system.
+^^^^^^^^^^^^^^^^^^^^^^
+
+Running apt-get command to update system:
+
+.. code-block:: 
+
+    sudo apt update
+    sudo apt upgrade
+
+Find out information about your GPU:
+
+.. code-block:: 
+
+    lspci | grep VGA
+
+.. image:: images/GPU-device.png
+  :width: 80%
+  :align: center
+
+2. Show all driver packages.
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To show all drvier packages which apply to the current system, please type the command:
+
+.. code-block:: 
+
+    sudo ubuntu-drivers devices
+
+Here is what we see:
+
+.. image:: images/ubuntu-driver-devices.png
+  :width: 80%
+  :align: center
+
+3. Install the GPU driver. 
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Here show two way how to install driver:
```

Comment:
Here show two ways how to install the driver:

Reply:
Done